react-with-graphql-tutorial.md

```diff
@@ -23,9 +23,9 @@ summary = "This React with GraphQL Tutorial shows you how to use GraphQL in your
 
 {{% read_before_2 "This tutorial is part 2 of 3 in this series." "Part 1:" "Getting Started with GitHub's GraphQL API" "https://www.robinwieruch.de/getting-started-github-graphql-api" "Part 3:" "A complete React with Apollo and GraphQL Tutorial" "https://www.robinwieruch.de/react-graphql-apollo-tutorial" %}}
 
-In the following client-sided GraphQL application that we are going to build together, you will learn how to combine React with GraphQL. There is no clever library such as {{% a_blank "Apollo Client" "https://github.com/apollographql/apollo-client" %}} or {{% a_blank "Relay" "https://github.com/facebook/relay" %}} involved yet to help you out in the beginning. Instead, you will perform GraphQL queries and mutations with basic HTTP requests. In a later GraphQL tutorial, it's planned to introduce Apollo as GraphQL client for your React.js application. But for now, this application should should only show you how to use GraphQL in React with HTTP. Yet you will start out even without React to learn just about GraphQL in the first place.
+In the  client-sided GraphQL application we'll build together, you will learn how to combine React with GraphQL. There is no clever library like {{% a_blank "Apollo Client" "https://github.com/apollographql/apollo-client" %}} or {{% a_blank "Relay" "https://github.com/facebook/relay" %}} to help yoy get started yet, so instead, you will perform GraphQL queries and mutations with basic HTTP requests. Later I'll introduce Apollo as a GraphQL client for your React.js application. For now, the application we build should should only show how to use GraphQL in React with HTTP. 
 
-Along the way of this React with GraphQL tutorial, you will build a simplified {{% a_blank "GitHub client" "https://github.com/rwieruch" %}}, basically an issue tracker for GitHub, that consumes {{% a_blank "GitHub's GraphQL API" "https://developer.github.com/v4/" %}}. You will perform GraphQL queries and mutations to read and write data. At the end of this GraphQL in React tutorial, you should be able to showcase a GraphQL in React example that can be used by other developers to learn from it. The final application you are going to build can be found in this {{% a_blank "repository on GitHub" "https://github.com/rwieruch/react-graphql-github-vanilla" %}}.
+Along the way, you will build a simplified {{% a_blank "GitHub client" "https://github.com/rwieruch" %}}, basically an issue tracker for GitHub, that consumes {{% a_blank "GitHub's GraphQL API" "https://developer.github.com/v4/" %}}. You will perform GraphQL queries and mutations to read and write data, and by the end, you should be able to showcase an example that can be used by other developers as a learning tool. The final application you are going to build can be found in this {{% a_blank "repository on GitHub" "https://github.com/rwieruch/react-graphql-github-vanilla" %}}.
```

Comment:
... to showcase a GraphQL in React example ... instead of ... to showcase an example ... because of better SEO

Reply:
"{{% a_blank "GitHub client" "https://github.com/rwieruch" %}}" to "GitHub client" <- remove link